conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="60" ground_alt="0" lat0="52.168595" lon0="4.412444" max_dist_from_home="1100" name="Rotating wing Valkenburg" security_height="2">
+  <header>
+    #include "autopilot.h"
+    #include "modules/datalink/datalink.h"
+</header>
```

Comment:
I don't think these includes are needed anymore. All headers included by modules are available here as well.

Reply:
Tried without and it works.